bindinfo/bind_test.go

```diff
@@ -1388,3 +1388,50 @@ func TestDropBindBySQLDigest(t *testing.T) {
 	tk.MustGetErrMsg(fmt.Sprintf("drop binding for sql digest '%s'", "1"), "can't find any binding for `1`")
 	tk.MustGetErrMsg(fmt.Sprintf("drop binding for sql digest '%s'", ""), "sql digest is empty")
 }
+
+func TestCreateBindingFromHistory(t *testing.T) {
```

Comment:
If the binding already exists, what will happen after create binding from history?

Reply:
will be covered, only the newest exists